app/client/stylesheets/_home.scss

```diff
@@ -73,32 +58,40 @@
 		position: absolute;
 	}
 
-	.logo-header {
-		height: 100px;
-		padding: 35px 0 0 50px;
-		position: absolute;
+	#hero-content {
```

Comment:
Is there a specific reason we use Id instead of a class for the hero-content?

Reply:
Originally for specificity reasons, but not anymore. Changed it into a class.